code/modules/food/recipes_microwave.dm

```diff
@@ -1747,30 +1747,74 @@
 		)
 	result = /obj/item/weapon/reagent_containers/food/snacks/flammkuchen
 
-// Currently Disabled //////////////////////////////////////////
```

Comment:
You should keep these in to avoid messing up the git history and to let us re-enable them in the future.